cnf-certification-test/certification/suite.go

```diff
@@ -202,3 +207,22 @@ func testContainerCertificationStatusByDigest(env *provider.TestEnvironment, val
 	}
 	testhelper.AddTestResultLog("Non-compliant", failedContainers, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+func testHelmVerion(env *provider.TestEnvironment) {
+	helmchartsReleases := env.HelmChartReleases
+	// Collect all of the failed helm charts
+	failedHelmCharts := [][]string{}
+	var helmChartVersionf float64
+	for _, helm := range helmchartsReleases {
+		helmChartVersion := helm.Chart.Metadata.APIVersion
+		helmChartVersion = strings.Split(helmChartVersion, "+")[0]
+		helmChartVersion = strings.Split(helmChartVersion, "v")[1]
+		helmChartVersionf, _ = strconv.ParseFloat(helmChartVersion, 64)
+
+		if helmChartVersionf > 0 && helmChartVersionf < 3 {
+			failedHelmCharts = append(failedHelmCharts, []string{helm.Chart.Metadata.Version, helm.Name})
+			tnf.ClaimFilePrintf("Helm Chart need to be v3 and not v2 which is not supported due to security risks associated with Tiller %s", helmChartVersionf)
```

Comment:
add new compliant/non compliant objects. Check this PR: https://github.com/test-network-function/cnf-certification-test/pull/1046. Create a new object type helmChart and a field name API version